src/main/java/org/fundacionjala/at15/pacman/Buttons.java

```diff
@@ -0,0 +1,56 @@
+package org.fundacionjala.at15.pacman;
+
+import javax.swing.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.awt.Color;
+
+public class Buttons {
```

Comment:
It would make sense to rename this class (example, ButtonsGroup) so it avoids redundancy with its contents (buttons)